cmd/promtool/main_test.go

```diff
@@ -60,6 +60,7 @@ func TestMain(m *testing.M) {
 }
 
 func TestQueryRange(t *testing.T) {
+	t.Parallel()
```

Comment:
In this test there are two informal test cases, you could build them using table tests and run them in parallel:

Suggestion:
```
func TestQueryRange(t *testing.T) {
	t.Parallel()
	testCases := []struct {
		name         string
		step         time.Duration
		assertResult func(t *testing.T, form url.Values, exitCode int)
	}{
		{
			name: "empty step",
			step: 0,
			assertResult: func(t *testing.T, form url.Values, exitCode int) {
				require.Equal(t, "up", form.Get("query"))
				require.Equal(t, "1", form.Get("step"))
				require.Equal(t, 0, exitCode)
			},
		},
		{
			name: "10 milliseconds step",
			step: 10 * time.Millisecond,
			assertResult: func(t *testing.T, form url.Values, exitCode int) {
				require.Equal(t, "up", form.Get("query"))
				require.Equal(t, "0.01", form.Get("step"))
				require.Equal(t, 0, exitCode)
			},
		},
	}
	for _, tt := range testCases {
		t.Run(tt.name, func(t *testing.T) {
			t.Parallel()
			s, getRequest := mockServer(200, `{"status": "success", "data": {"resultType": "matrix", "result": []}}`)
			defer s.Close()

			urlObject, err := url.Parse(s.URL)
			require.NoError(t, err)

			p := &promqlPrinter{}
			exitCode := QueryRange(urlObject, http.DefaultTransport, map[string]string{}, "up", "0", "300", tt.step, p)
			require.Equal(t, "/api/v1/query_range", getRequest().URL.Path)
			form := getRequest().Form
			tt.assertResult(t, form, exitCode)
		})
	}
}
```

Reply:
I want to keep the changes minimal. Since there are only three tests, seems alright to add `t.Parallel()` in each of them.